doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,416 @@
+spd10max,Hourly maximum wind speed
+t02max,Max hourly temperature at 2 meters
+t02min,Min hourly temperature at 2 meters
+rh02max,Max hourly relative humidity at 2 meters
+rh02min,Min hourly relative humidity at 2 meters
+pratemax,Max hourly precipitation rate
+frzr,Accumulated surface freezing rain
+frzrb,Accumulated surface freezing rain in bucket
+frozr,Accumulated surface graupel
+frozrb,Accumulated surface graupel in bucket
+tsnowp,Accumulated surface snow
+tsnowpb,Accumulated surface snow in bucket
+rhonewsn,Precipitation ice density
+rain,Total rain at this time step
+rainc,Convective rain at this time step
+ice,Ice fall at this time step
+snow,Snow fall at this time step
+graupel,Graupel fall at this time step
+totice_ave,Surface ice precipitation rate
+toticeb_ave,Bucket surface ice precipitation rate
+totsnw_ave,Surface snow precipitation rate
+totsnwb_ave,Bucket surface snow precipitation rate
+totgrp_ave,Surface graupel precipitation rate
+totgrpb_ave,Bucket surface graupel precipitation rate
+u10m,10 meter u wind
+v10m,10 meter v wind
+dpt2m,2 meter dewpoint temperature
+hgt_hyblev1,Layer 1 height
+psurf,Surface pressure
+hpbl,Surface planetary boundary layer height
+pwat,Atmosphere column precipitable water
+tmp_hyblev1,Layer 1 temperature
+spfh_hyblev1,Layer 1 specific humidity
+ugrd-hyblev1,Layer 1 zonal wind
+vgrd-hyblev1,Layer 1 meridional wind
+sfexc,Exchange coefficient
+acond,Aerodynamic conductance
+dlwsfci,Instantaneous surface downward longwave flux
+ulwsfci,Instantaneous surface upward longwave flux
+dswsfci,Instantaneous surface downward shortwave flux
+uswsfci,Instantaneous surface upward shortwave flux
+dusfci,Instantaneous u component of surface stress
+dvsfci,Instantaneous v component of surface stress
+shtfl,Instantaneous surface sensible heat net flux
+lhtfl,Instantaneous surface latent heat net flux
+gfluxi,Instantaneous surface ground heat flux
+wilt,Wiltimg point (volumetric)
+fldcp,Field capacity (volumetric)
+pahi,Instantaneous precipitation advected heat flux
+pevpr,Instantaneous surface potential evaporation
+wet1,Normalized soil wetness
+cpofp,Precent frozen precipitation
+crain_ave,Averaged categorical rain
+csnow_ave,Averaged categorical snow
+cfrzr_ave,Averaged categorical freezing rain
+cicep_ave,Averaged categorical sleet
+refl_10cm,Radar reflectivity
+max_hail_diam_sfc,Maximum hail diameter at lowest model level
+dkt,Atmospheric heat diffusivity
+dku,Atmospheric momentum diffusivity
+cldfra,Instantaneous 3D cloud fraction
+cnvw,Subgrid scale convective cloud water
+skebu_wts,Perturbation velocity - u component
+skebv_wts,Perturbation velocity - v component
+zmtnblck,Level of dividing streamline
+refdmax,Max hourly 1-kilometer above ground level reflectivity
+refdmax263k,Max hourly -10C reflectivity
+sppt_wts,Perturbation velocity
+shum_wts,Perturbation velocity
+spp_wts_pbl,Stochastic perturbed parameterization planetary boundary layer perturbation wind turbines

```

Comment:
```suggestion
spp_wts_pbl,Stochastically perturbed parameterization - planetary boundary layer
```

Reply:
@selbigmtnwx23 I'll see if I can confirm what wts stands for in these variables. If it's wind turbines, I'll add it back in, but I did want to check.

Reply:
Again, maybe its wrong of me to be getting these variables from research artciles. But WTs in research on stochastic simulations stands for wind turbines. chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://hal.science/hal-01809938/file/Stochasticsimulationsandstabilitytodeterminemaximumwindpowerpenetrationofanislandnetwork.pdf

Reply:
I have confirmed with Jeff Beck, who implemented these fields, that wts stands for "weights," aka the magnitude of the perturbation.

Reply:
Sounds good, thanks for checking!